addons/web_editor/static/lib/odoo-editor/test/spec/copyPaste.test.js

```diff
@@ -1232,11 +1232,11 @@ describe('Copy and paste', () => {
             });
             it('should paste and transform an URL in a span', async () => {
                 await testEditor(BasicEditor, {
-                    contentBefore: '<p>a<span>b[]c</span>d</p>',
+                    contentBefore: '<p>a<span odoo-test-ignore-attr>b[]c</span>d</p>',
                     stepFunction: async editor => {
                         await pasteText(editor, 'http://www.xyz.com');
                     },
-                    contentAfter: '<p>a<span>b<a href="http://www.xyz.com">http://www.xyz.com</a>[]c</span>d</p>',
+                    contentAfter: '<p>ab<a href="http://www.xyz.com">http://www.xyz.com</a>[]cd</p>',
```

Comment:
@dmo-odoo For illustration, if I were to call `setSelection` here, it would move my selection within the `a`.

Reply:
But why though ? Wouldn't we want `setSelection` to set the selection precisely where we ask it to ? Ah... maybe not because of `normalize = true`... Is this what you mean ?